apps/chrome-devtools/src/extension/background.ts

```diff
@@ -1,12 +1,18 @@
-import type { InjectContentMessage, OtterMessage, otterMessageType } from '@o3r/core';
-import type { scriptToInject as ScriptToInject } from '../services/connection.service';
-import type { ExtensionMessage } from './interface';
+import type { InjectContentMessage, OtterMessage, OtterMessageContent, otterMessageType } from '@o3r/core';
+import type { ApplicationInformationContentMessage } from '@o3r/application';
+import { type scriptToInject as ScriptToInject } from '../services/connection.service';
```

Comment:
```suggestion
import type { scriptToInject as ScriptToInject } from '../services/connection.service';
```

do not change anything but accentuate the point that only `import type` is allow in this file